packages/website/utils/content.ts

```diff
@@ -64,7 +64,7 @@ async function getAllMdx(paths: string[]) {
 const allMdxSources = [
   path.resolve('../../packages/components'),
   path.resolve('../../packages/core'),
-  path.resolve('../../packages/forma-36-website/src/content'),
+  // path.resolve('../../packages/forma-36-website/src/content'),
```

Comment:
We temporary disable picking up content from Gatsby website, because each item will be copied later one by one.

Reply:
Maybe add this comment to the code, wdyt?